examples/tutorials/advance_model_config/02_device.ipynb

```diff
@@ -0,0 +1,489 @@
+{
```

Comment:
<p>In TF tutorial it is written that: "By default, batchflow pipeline models use all available GPUs. "</p><p>Ensure that in all tutorials you write the same, preferably what is true</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/analysiscenter/batchflow/pull/332/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>